File: 1984_Minimum_Difference_Between_Highest_and_Lowest_of_K_Scores/solution.c
#include <stdlib.h>

int compare(const void* a, const void* b) { return (*(int*)a - *(int*)b); }
int minimumDifference(int* nums, int numsSize, int k) {
    qsort(nums, numsSize, sizeof(int), compare);
    int result = nums[k - 1] - nums[0];
    for (int i = 0; i + k <= numsSize; i++) result = result > nums[i + k - 1] - nums[i] ? nums[i + k - 1] - nums[i] : result;
    return result;
}

File: 1984_Minimum_Difference_Between_Highest_and_Lowest_of_K_Scores/solution.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        n = len(nums)
        nums.sort()
        result = nums[k - 1] - nums[0]
        for i in range(n - k + 1):
            result = min(result, nums[i + k - 1] - nums[i])
        return result